HW4/OleksandrPanchuk/task.py

```diff
@@ -0,0 +1,6 @@
+Celsius = float(input("Enter the temperature in Celsius: "))
```

Comment:
good job
I recommend using lowercase variable names like "celsius"

Reply:
ok thanks for the advice